validation-test/execution/interpret-with-dependencies-unix.swift

```diff
@@ -1,4 +1,4 @@
-// REQUIRES: OS=linux-gnu
+// UNSUPPORTED: OS=macosx || OS=tvos || OS=ios || OS=watchos || OS=windows-msvc
```

Comment:
Now enabled this one failing test on the linux CI for all Unix platforms, ie including the BSDs and Android.